src/marlin_stubs/pause/pause.cpp

```diff
@@ -97,13 +94,28 @@ Pause &Pause::GetInstance() {
     return s;
 }
 
-//xyze_pos_t Pause::resume_position;
+void Pause::SetUnloadLenght(float len) {
+    unload_length = -ABS(len); // it is negative value
```

Comment:
use std functions in c++ (std::abs), no need to use the c-style 3rd party workarounds

Reply:
thx
fixed